### PR TITLE
Fix: Remove the attributes that facilitate address test cases only in consignment trac

### PR DESCRIPTION
**Pull request by Jerify02**
Close #6472 .

### CODE REVIEW

File: projects/storefrontlib/src/cms-components/myaccount/order/order-details/order-detail-items/consignment-tracking/consignment-tracking.component.spec.ts [hunk 1]
```diff
@@ -140,7 +140,7 @@ describe('ConsignmentTrackingComponent', () => {
     mockConsignment.status = consignmentStatus[0];
     fixture.detectChanges();
 
-    el.query(By.css('[data-test="btn-events"]')).nativeElement.click();
+    el.query(By.css('.bth-track')).nativeElement.click();
```
Comment (marlass) [inline projects/storefrontlib/src/cms-components/myaccount/order/order-details/order-detail-items/consignment-tracking/consignment-tracking.component.spec.ts]:
```suggestion
    el.query(By.css('.btn-track')).nativeElement.click();
```

File: projects/storefrontlib/src/cms-components/myaccount/order/order-details/order-detail-items/consignment-tracking/consignment-tracking.component.spec.ts [hunk 2]
```diff
@@ -121,14 +121,14 @@ describe('ConsignmentTrackingComponent', () => {
...
-    expect(el.query(By.css('[data-test="btn-events"]'))).toBeFalsy();
+    expect(el.query(By.css('.bth-track'))).toBeFalsy();
   });
 
   it('should display tracking package button', () => {
     consignmentStatus.forEach(status => {
       mockConsignment.status = status;
       fixture.detectChanges();
-      expect(el.query(By.css('[data-test="btn-events"]'))).toBeTruthy();
+      expect(el.query(By.css('.bth-track'))).toBeTruthy();
```
Comment (marlass) [inline projects/storefrontlib/src/cms-components/myaccount/order/order-details/order-detail-items/consignment-tracking/consignment-tracking.component.spec.ts]:
```suggestion
      expect(el.query(By.css('.btn-track'))).toBeTruthy();
```

File: projects/storefrontlib/src/cms-components/myaccount/order/order-details/order-detail-items/consignment-tracking/consignment-tracking.component.spec.ts [hunk 3]
```diff
@@ -121,14 +121,14 @@ describe('ConsignmentTrackingComponent', () => {
       arrayEqyals(consignmentStatus, component.consignmentStatus)
     ).toBeTruthy();
     expect(consignmentStatus.includes(status)).toBeFalsy();
-    expect(el.query(By.css('[data-test="btn-events"]'))).toBeFalsy();
+    expect(el.query(By.css('.bth-track'))).toBeFalsy();
```
Comment (marlass) [inline projects/storefrontlib/src/cms-components/myaccount/order/order-details/order-detail-items/consignment-tracking/consignment-tracking.component.spec.ts]:
```suggestion
    expect(el.query(By.css('.btn-track'))).toBeFalsy();
```

File: projects/storefrontlib/src/cms-components/myaccount/order/order-details/order-detail-items/consignment-tracking/consignment-tracking.component.spec.ts [hunk 4]
```diff
@@ -111,7 +111,7 @@ describe('ConsignmentTrackingComponent', () => {
...
```
Comment (marlass) [inline projects/storefrontlib/src/cms-components/myaccount/order/order-details/order-detail-items/consignment-tracking/consignment-tracking.component.spec.ts]:
```suggestion
    expect(el.query(By.css('.btn-track'))).toBeFalsy();
```

File: projects/storefrontlib/src/cms-components/myaccount/order/order-details/order-detail-items/consignment-tracking/consignment-tracking.component.spec.ts [hunk 5]
```diff
@@ -100,7 +100,7 @@ describe('ConsignmentTrackingComponent', () => {
...
```
Comment (marlass) [inline projects/storefrontlib/src/cms-components/myaccount/order/order-details/order-detail-items/consignment-tracking/consignment-tracking.component.spec.ts]:
```suggestion
    expect(el.query(By.css('.btn-track'))).toBeFalsy();
```

File: projects/storefrontlib/src/cms-components/myaccount/order/order-details/order-detail-items/consignment-tracking/consignment-tracking.component.html [hunk 6]
```diff
@@ -2,8 +2,7 @@
   <div *ngIf="consignmentStatus.includes(consignment.status)">
     <button
       (click)="openTrackingDialog(consignment)"
-      class="btn btn-action"
-      data-test="btn-events"
+      class="btn btn-action bth-track"
```
[the remark this answers is not in the document]
Comment (Jerify02) [reply projects/storefrontlib/src/cms-components/myaccount/order/order-details/order-detail-items/consignment-tracking/consignment-tracking.component.html]: fixed.

File: projects/storefrontapp-e2e-cypress/cypress/helpers/consignment-tracking.ts [hunk 7]
```diff
@@ -53,5 +53,5 @@ export function verifyTrackingBtn() {
 }
 
 export function verifyNoTrackingBtn() {
-  cy.get('[data-test="btn-events"]').should('have.length', 0);
+  cy.get('.bth-track').should('have.length', 0);
```
[the remark this answers is not in the document]
Comment (Jerify02) [reply projects/storefrontapp-e2e-cypress/cypress/helpers/consignment-tracking.ts]: fixed.

File: projects/storefrontapp-e2e-cypress/cypress/helpers/consignment-tracking.ts [hunk 8]
```diff
@@ -32,17 +32,17 @@ export function verifyTrackingBtn() {
...
     });
-  cy.get('[data-test="body-event"]').should('have.length', 3);
+  cy.get('.event-body').should('have.length', 3);
   cy.get('.close').click();
   cy.get('.cx-list')
     .next()
     .within(() => {
       cy.get('.cx-code').should('contain', '1992693');
-      cy.get('[data-test="btn-events"]').click();
+      cy.get('.bth-track').click();
```
[the remark this answers is not in the document]
Comment (Jerify02) [reply projects/storefrontapp-e2e-cypress/cypress/helpers/consignment-tracking.ts]: fixed.

File: projects/storefrontapp-e2e-cypress/cypress/helpers/consignment-tracking.ts [hunk 9]
```diff
@@ -32,17 +32,17 @@ export function verifyTrackingBtn() {
     .first()
     .within(() => {
       cy.get('.cx-code').should('contain', '300938');
-      cy.get('[data-test="btn-events"]').click();
+      cy.get('.bth-track').click();
```
[the remark this answers is not in the document]
Comment (Jerify02) [reply projects/storefrontapp-e2e-cypress/cypress/helpers/consignment-tracking.ts]: fixed.

File: projects/storefrontlib/src/cms-components/myaccount/order/order-details/order-detail-items/consignment-tracking/consignment-tracking.component.spec.ts (hunk 1 above)
Comment (Jerify02) [reply projects/storefrontlib/src/cms-components/myaccount/order/order-details/order-detail-items/consignment-tracking/consignment-tracking.component.spec.ts]: fixed.

File: projects/storefrontlib/src/cms-components/myaccount/order/order-details/order-detail-items/consignment-tracking/consignment-tracking.component.spec.ts (hunk 2 above)
Comment (Jerify02) [reply projects/storefrontlib/src/cms-components/myaccount/order/order-details/order-detail-items/consignment-tracking/consignment-tracking.component.spec.ts]: fixed.

File: projects/storefrontlib/src/cms-components/myaccount/order/order-details/order-detail-items/consignment-tracking/consignment-tracking.component.spec.ts (hunk 3 above)
Comment (Jerify02) [reply projects/storefrontlib/src/cms-components/myaccount/order/order-details/order-detail-items/consignment-tracking/consignment-tracking.component.spec.ts]: fixed.

File: projects/storefrontlib/src/cms-components/myaccount/order/order-details/order-detail-items/consignment-tracking/consignment-tracking.component.spec.ts (hunk 4 above)
Comment (Jerify02) [reply projects/storefrontlib/src/cms-components/myaccount/order/order-details/order-detail-items/consignment-tracking/consignment-tracking.component.spec.ts]: fixed.

File: projects/storefrontlib/src/cms-components/myaccount/order/order-details/order-detail-items/consignment-tracking/consignment-tracking.component.spec.ts (hunk 5 above)
Comment (Jerify02) [reply projects/storefrontlib/src/cms-components/myaccount/order/order-details/order-detail-items/consignment-tracking/consignment-tracking.component.spec.ts]: fixed.

**Pull request merged**